app/controllers/application_controller.rb

```diff
@@ -1,2 +1,3 @@
 class ApplicationController < ActionController::Base
+  default_form_builder GOVUKDesignSystemFormBuilder::FormBuilder
```

Comment:
Would expect this to be in a `config/initializer` rather than the base controller?

Reply:
Was following [the instructions on the form builder site](https://govuk-form-builder.netlify.app/introduction/getting-started/), but more than happy to move to a different location if you can show me how/where 🙂

Reply:
Appears I'm wrong and this is now the recommended place to put it. Nice :)